TimeTable/TimeTable/Program.cs

```diff
@@ -0,0 +1,152 @@
+                case "c":
+                    AddEvents();
+                    break;
+                case "D":
+                    CloseCalendar();
+                    break;
+            }
+        }
+
+        static void ShowAllEvents()
+        {
+            Console.WriteLine();
+            Console.WriteLine("Here is a list of saved events: ");
+            Database databaseObject = new Database();
+            
+            // View all saved events
+            string queryAllEvents = "SELECT * FROM myCalendar ORDER BY eventDate ASC";
+            SQLiteCommand selectAllCommand = new SQLiteCommand(queryAllEvents, databaseObject.myConnection);
+            databaseObject.OpenConnection();
+            SQLiteDataReader resultAll = selectAllCommand.ExecuteReader();
+            if (resultAll.HasRows)
+            {
+                while (resultAll.Read())
+                {
+                    Console.WriteLine("Event Data: {0} - Event Name: {1}", resultAll["eventDate"], resultAll["eventName"]);
+                }
+            }
+            ShowMenu();
+        }
+
+        // Display selected date
+        static void SelectOneDay()
+        {
+            // Ask the user to enter any date
+            string enteredDay;
+            Console.WriteLine("Enter the date in the format YYYY-MM-DD: ");
+            enteredDay = Console.ReadLine();
+
+            ShowEventsByDay(enteredDay);
+            ShowMenu();
+        }
+
+        static void ShowEventsByDay(string enteredDay)
+        {
+            // Search and display the selected date
+            Database databaseObject = new Database();
+
+            string query = $"SELECT * FROM myCalendar WHERE eventDate='{enteredDay}'";
+
+            SQLiteCommand selectOneDayCommand = new SQLiteCommand(query, databaseObject.myConnection);
+            databaseObject.OpenConnection();
+            SQLiteDataReader resultFromOneDay = selectOneDayCommand.ExecuteReader();
+            if (resultFromOneDay.HasRows)
+            {
+                while (resultFromOneDay.Read())
+                {
+                    Console.WriteLine("Event Data: {0} - Event Name: {1}", resultFromOneDay["eventDate"], resultFromOneDay["eventName"]);
+                }
+            }
+            else
+            {
+                Console.WriteLine();
+                Console.WriteLine("You have no events for this day");
+            }
+            
+         }
+
+        static void AddEvents()
+        {
+            // Add event to The calendar
+            Database databaseObject = new Database();
+
+            string query = "INSERT INTO myCalendar (`eventDate`, `eventName`) VALUES (@eventDate, @eventName)";
+            SQLiteCommand myCommand = new SQLiteCommand(query, databaseObject.myConnection);
+
+            databaseObject.OpenConnection();
+
+            // Ask the user to enter a date 
+            string eventDate;
+            Console.WriteLine("Enter the day in the format YYYY-MM-DD: ");
```

Comment:
It would be nice to create some kind of validation to check if date was inputted correctly.

Reply:
Validation has been implemented
`        // Display selected date
        static void SelectOneDay()
        {
            // Ask the user to enter any date
            string enteredDay;
            DateTime dateValue;

            Console.WriteLine("Enter the date in the format YYYY-MM-DD: ");
            enteredDay = Console.ReadLine();

            if (DateTime.TryParse(enteredDay, out dateValue))
            {
                ShowEventsByDay(enteredDay);
                ShowMenu();
            }
            else
                Console.WriteLine("The date format is incorrect", enteredDay);
                Console.WriteLine();
                Console.WriteLine("You want to enter the date again? Y/N: ");
                switch (Console.ReadLine().ToUpper())
                {
                case "Y":
                    SelectOneDay();
                    break;
                case "N":
                    ShowMenu();
                    break;
                }
        }
`